libs/sdk-ui-charts/src/highcharts/chartTypes/_chartCreators/customConfiguration.ts

```diff
@@ -518,6 +518,7 @@ function getTooltipConfiguration(
                   outside: true,
                   positioner: partial(getTooltipPositionInViewPort, chartType, stacking),
                   formatter: partial(formatTooltip, tooltipAction, intl),
+                  enabled: _chartConfig?.tooltip?.enabled !== undefined ? _chartConfig.tooltip.enabled : true,
```

Comment:
You could make this simpler by `enabled: _chartConfig?.tooltip?.enabled ?? true`. This will do the same thing as your code.